Makefile

```diff
@@ -33,11 +33,11 @@ export GO111MODULE=on
 # Directories.
 ROOT_DIR:=$(shell dirname $(realpath $(firstword $(MAKEFILE_LIST))))
 TOOLS_DIR := hack/tools
-TOOLS_BIN_DIR := $(TOOLS_DIR)/bin
+TOOLS_BIN_DIR := $(abspath $(TOOLS_DIR)/bin)
```

Comment:
macOS does not have `abspath` out of the box - if you need this, I would do `cd $(TOOLS_DIR)/bin && pwd -P`

Reply:
I actually built this on OS X. 

```
$ make --version
GNU Make 3.81
Copyright (C) 2006  Free Software Foundation, Inc.
This is free software; see the source for copying conditions.
There is NO warranty; not even for MERCHANTABILITY or FITNESS FOR A
PARTICULAR PURPOSE.

This program built for i386-apple-darwin11.3.0
```

Reply:
My bad - was thinking this was the shell abspath, not the make built-in.